src/scss/2022/components/_hero.scss

```diff
@@ -24,7 +25,11 @@
     font-family: var(--font-display);
     grid-area: b;
     // Allows the font to be fluid in relation to the hero size
-    font-size: min(14vw, calc(var(--hero-max-width) * 0.14));
+    font-size: min(12vw, calc(var(--hero-max-width) * 0.12));
```

Comment:
Where's 0.12 coming from? (same question for all these)

Reply:
Percentages! So 12vw is 12% of the width of the screen, so the `0.12` is 12% of `var(--hero-max-width)`. It keeps the font-size from growing past 12% of the hero’s `max-width`

Reply:
Ohh of course!! Nice. Thank you!